### PR TITLE
issue #312: stepper & validation forms integration

### PR DESCRIPTION
**Pull request by k-allagbe**
- [x] LabelData: represents the data extracted from the pipeline
- [x] the forms will share the label data via the FormComponentProps interface
- [x] tests

### CODE REVIEW

File: src/components/DummyStepComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { StepComponentProps } from "./stepper";
+
+function DummyStepComponent({ title }: StepComponentProps) {
+  return <div data-testid={title}>{title}</div>;
+}
+
+export default DummyStepComponent;
```
Comment (ChromaticPanic) [inline src/components/DummyStepComponent.tsx:7]: Are you using this in multiple tests? Might be better to move this definition inside the test file

File: src/types/Alert.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+export type AlertSeverity = OverridableStringUnion<
+  AlertColor,
+  AlertPropsColorOverrides
+>;
+
+export default interface Alert {
+  message: string;
+  type: AlertSeverity;
+}
```
Comment (ChromaticPanic) [inline src/types/Alert.ts:13]: You may want to just combine all these types files int a single types.d.ts file. Or keep these interface definitions with the components that they are used with. I don't think this project will evolve into something that has too many types for a single type definition file.

**Review by ChromaticPanic — CHANGES_REQUESTED** [review]

File: src/components/DummyStepComponent.tsx (hunk 1 above)
Comment (k-allagbe) [reply src/components/DummyStepComponent.tsx]: It is also there for manual testing, like the upload images button. I will replace it with the real forms.

File: src/types/Alert.ts (hunk 2 above)
Comment (k-allagbe) [reply src/types/Alert.ts]: Done. Although the .d.ts file is causing some issues in tests so I just used a .ts file. I must admit that I'm not too familiar with .d.ts

**Pull request merged**